stacks/shared-kinesis.yml

```diff
@@ -0,0 +1,90 @@
+# stacks/shared-kinesis.yml
+AWSTemplateFormatVersion: "2010-09-09"
+
+Description: >-
+  Creates kinesis streams intended to be used by several applications.
+
+Parameters:
+  EnvironmentType: { Type: String }
+  RootStackName: { Type: String }
+  RootStackId: { Type: String }
+
+Resources:
+  # dovetail3-cdn realtime logs --> dovetail-counts
+  CountsBytesKinesisStream:
```

Comment:
Threw some comments in here, describing the inputs/outputs of each stream.

I'm not entirely sold on the naming.  But tried to reuse the existing manually-created-stream names, and be consistent throughout every file.  In general, the stream name describes which lambda(s) it _triggers_.